### DISCUSSION TITLE
Looking for a case study to feature

### DISCUSSION THREAD

**Discussion by jessicadunbar** [Leantime/leantime#1370] [Show and tell]
Hello Community, I'm writing an article for opensource.com and am looking for a case study or a brand that is using your software.  For example:
The Department of Homeland Security needed a highly secure messaging platform to communicate across US government agencies. Rocket.Chat Satisfied the requirements.  

Public Sector would be nice but not required. 
Thank you!

**Comment by Chromico** [comment]
At thundergear.io we needed a project management tool that was easy to use, reliable, open source, and could be securely hosted on our cloud providers. Leantime exceeded those requirements and made organizing our projects as simple as a walk in the park.

**Comment by marcelfolaron** [comment]
Thank you @jessicadunbar. I posted your request on our discord channel as well and there are 2 additional companies that replied. https://discord.gg/4zMzJtAq9z
